openmc/mgxs/mgxs.py

```diff
@@ -756,12 +770,19 @@ def get_xs(self, groups='all', subdomains='all', nuclides='all',
         cv.check_value('value', value, ['mean', 'std_dev', 'rel_err'])
         cv.check_value('xs_type', xs_type, ['macro', 'micro'])
 
+        # FIXME: Unable to get microscopic xs for mesh domain because the mesh
+        # cells do not know the nuclide densities in each mesh cell.
+        if self.domain_type == 'mesh' and xs_type == 'micro':
+            msg = 'Unable to get micro xs for mesh domain since the mesh ' \
+                  'cells do not know the nuclide densities in each mesh cell.'
```

Comment:
And I should say -- the capability is both for getting the volume of an arbitrary region as well as average nuclide densities, the latter being what you need.